sharding/nodesCoordinator/hashValidatorShuffler.go

```diff
@@ -18,13 +18,10 @@ var _ NodesShuffler = (*randHashShuffler)(nil)
 
 // NodesShufflerArgs defines the arguments required to create a nodes shuffler
 type NodesShufflerArgs struct {
-	NodesShard           uint32
-	NodesMeta            uint32
-	Hysteresis           float32
-	Adaptivity           bool
-	ShuffleBetweenShards bool
-	MaxNodesEnableConfig []config.MaxNodesChangeConfig
-	EnableEpochsHandler  common.EnableEpochsHandler
+	ShuffleBetweenShards   bool
+	MaxNodesEnableConfig   []config.MaxNodesChangeConfig
+	EnableEpochsHandler    common.EnableEpochsHandler
+	ChainParametersHandler ChainParametersHandler
```

Comment:
check also if the notify order for chain parameters handler.
it needs to at least be before nodes coordinator.